### ISSUE TITLE
selecting "filter this post" causes crash

### ISSUE THREAD

**Issue by pmes** [glitch-soc/mastodon#2186]
### Steps to reproduce the problem

regardless of context

click the elipsis menu for a post 

select "filter this post" from the menu

### Expected behaviour

open filter creation dialog

### Actual behaviour

crashes every time

### Detailed description

Stack trace
-----------

```
Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7B%24%24typeof%2C%20type%2C%20compare%2C%20WrappedComponent%2C%20displayName%2C%20statusId%2C%20onChangeBackgroundColor%2C%20onClose%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
xr@https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:265964
jr/<@https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:270822
Bo@https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:280233
ws@https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:318741
mu@https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:311088
_u@https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:311011
su@https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:308041
Xi/<@https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:259789
t.unstable_runWithPriority@https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:334671
$i@https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:259498
Xi@https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:259736
Yi@https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:259669
au@https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:304826
enqueueSetState@https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:263454
y.prototype.setState@https://assets.infosec.exchange/packs/js/common-7b96bacb9789a1dd3e57.chunk.js:2:209393
681/c/this.load/<@https://assets.infosec.exchange/packs/js/334-7eb0688aad5445721be4.chunk.js:2:151230

```Mapped stack trace
-----------

```
Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=object%20with%20keys%20%7B%24%24typeof%2C%20type%2C%20compare%2C%20WrappedComponent%2C%20displayName%2C%20statusId%2C%20onChangeBackgroundColor%2C%20onClose%7D&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
b (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:2658:8)
d (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:2885:127)
Error(u (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:3490:34)
L (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:5441:8)
Bg (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:5025:12)
Ag (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:5021:)
yj (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:4806:12)
gg (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:2318:4)
unstable_runWithPriority (webpack:///node_modules/scheduler/cjs/scheduler.production.min.js:238:)
bg (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:2290:9)
eg (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:2311:49)
dg (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:2308:12)
Ig (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:4647:11)
enqueueSetState (webpack:///node_modules/react-dom/cjs/react-dom.production.min.js:2564:19)
F (webpack:///node_modules/react/cjs/react.production.min.js:45:12)
this.setState (webpack:///app/javascript/home/mastodon/live/app/javascript/flavours/glitch/features/ui/components/bundle.jsx:80:13)
```


React component stack
---------------------

```

    in Unknown
    in c
    in Connect(c)
    in div
    in div
    in div
    in f
    in De
    in Connect(De)
    in div
    in div
    in t
    in t
    in kt
    in t
    in withRouter(kt)
    in InjectIntl(withRouter(kt))
    in Connect(InjectIntl(withRouter(kt)))
    in t
    in t
    in t
    in withRouter(t)
    in t
    in t
    in Rt
    in c
    in IntlProvider
    in Kt
```

### Specifications

this is on infosec.exchange running mastodon v4.1.1+glitch

firefox 112

**Comment by Plastikmensch** [comment]
Oops, missed that.
The export default statement in filter_modal.jsx is incorrect.

**Comment by ClearlyClaire** [comment]
Fixed by #2192
